pkg/config/env/filter.go

```diff
@@ -9,3 +9,11 @@ const defaultConfigurationFileName = "filter.conf"
 func FilterConfigurationFileName() string {
 	return envy.Get("ATHENS_FILTER_FILENAME", defaultConfigurationFileName)
 }
+
+// FilterOff checks PROXY_FILTER_OFF env and returns
+// true of it's equal to "true", otherwise false always.
+// It defaults to "true" until Olympus is the default
+// place to grab modules before the Proxy.
+func FilterOff() bool {
+	return envy.Get("PROXY_FILTER_OFF", "true") == "true"
```

Comment:
Should the envy call default to `false` as the check is whether the filter is off?

like `envy.Get("PROXY_FILTER_OFF", "false") == "true"`
if `PROXY_FILTER_OFF` was `true`, result would be `true`
if `PROXY_FILTER_OFF` was 'false' result would be `false`

the way it is now, with default of true, if the user does not set `PROXY_FILTER_OFF` the result will be `true`.

Reply:
I guess not, re-reading the comment clears that up. :D

Reply:
Yes by default, it's just the Proxy, no filtering needed. But we will use Olympus by default once Olympus is ready.